pkgs/by-name/ma/marcel/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, python3
+, bash
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "marcel";
+  version = "0.20.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "geophile";
+    repo = "marcel";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DlpjE0RTn2zdTkSKXGvuZXBk8R6ZKrhRe8U7xchyauo=";
+  };
+
+  propagatedBuildInputs = [
+    bash
+  ] ++ (with python3.pkgs; [
+    dill
+    psutil
+    setuptools
+  ]);
+
+  # The tests use sudo and try to read/write $HOME/.local/share/marcel and /tmp
+  doCheck = false;
+
+  postFixup = ''
+    substituteInPlace $out/bin/marcel \
+        --replace python3 ${python3}/bin/python3
```

Nit: use only two spaces to indent.
```suggestion
      --replace python3 ${python3}/bin/python3
```

---

https://github.com/NixOS/nixpkgs/pull/267461#discussion_r1396529889

should i use two space indention?

---

yes, always 2 spaces in nix.